src/west/commands/project.py

```diff
@@ -711,11 +712,21 @@ def update(self, project):
             _clone(project)
 
         if self.fs == 'always' or _rev_type(project) not in ('tag', 'commit'):
-            new_manifest_rev = _fetch(project)
```

Comment:
Looks good! Almost there, thanks for all your patience.

Just one small change: now that the return value for _fetch() is being thrown away, I think `_fetch()` itself does not need to return anything. Other than that this looks great, thank you again for your work on this.

Reply:
good catch.
Fixed.